### PR TITLE
Doc/4547 keyword conventions

### PR DESCRIPTION
**Pull request by dcrawforAtAnsys**
Update Contributing.rst

### CODE REVIEW

File: doc/source/Getting_started/Contributing.rst [hunk 1]
```diff
@@ -73,6 +74,57 @@ For example::
...
+Naming conventions
+~~~~~~~~~~~~~~~~~~
+Consistency of names helps improve readability and
+ease of use. Starting with release 0.8 a concerted effort
+has been made to
+improve consistency of naming and adherence to
+:ref:`PEP-8<https://peps.python.org/pep-0008/>`_.
+
+For example, methods used to create or access entities in
+Electronics Desktop require that a name to be passed to the method or function
```
Comment (PipKat) [inline doc/source/Getting_started/Contributing.rst:86]:
```suggestion
AEDT require that a name be passed to the method or function
```

File: doc/source/Getting_started/Contributing.rst [hunk 2]
```diff
@@ -73,6 +74,57 @@ For example::
...
+as an argument.
+It is tempting to
+include context as part of that variable name. For example, one is tempted to use
+``setupname``
+as an argument to :meth:`Hfss.create_setup`_.
+However, the context "setup" is
+explicitly defined by the method name. The variable ``name`` provides
+a more compact
+description of the variable in this context.
```
Comment (PipKat) [inline doc/source/Getting_started/Contributing.rst:95]:
```suggestion
include context as part of that variable name. For example, while it is tempting to use
``setupname``
as an argument to :meth:`Hfss.create_setup`_,
the context "setup" is
explicitly defined by the method name. The variable ``name`` provides
a more compact
description of the variable in this context.
```

File: doc/source/Getting_started/Contributing.rst [hunk 3]
```diff
@@ -73,6 +74,57 @@ For example::
...
+
+One may
+also find both ``setup_name`` and ``setupname`` used in previous versions of PyAEDT
+for various methods or classes.
```
Comment (PipKat) [inline doc/source/Getting_started/Contributing.rst:99]:
```suggestion
In previous PyAEDT versions, you can also find both ``setup_name`` and ``setupname`` used 
for various methods or classes.
```

File: doc/source/Getting_started/Contributing.rst [hunk 4]
```diff
@@ -73,6 +74,57 @@ For example::
...
+Improving naming consistency improves maintainability and readability.
+
+The following table illustrates the recommended conventions:
+
+.. list-table:: Keywords and Object Names
```
Comment (PipKat) [inline doc/source/Getting_started/Contributing.rst:104]:
```suggestion
.. list-table:: Keywords and object names
```
Sentence case for all headings/tites.

File: doc/source/Getting_started/Contributing.rst [hunk 5]
```diff
@@ -73,6 +74,57 @@ For example::
...
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Old Name
+     - New Name
```
Comment (PipKat) [inline doc/source/Getting_started/Contributing.rst:109]:
```suggestion
   * - Old name
     - New name
```

File: doc/source/Getting_started/Contributing.rst [hunk 6]
```diff
@@ -73,6 +74,57 @@ For example::
...
+   * - ``entities``
+     - ``assignment``
+     - ``Maxwell.assign_current_density()``
+   * - ``entity_list``
+     - ``assignment``
+     - ``Maxwell.assign_symmetry()``
+
+Please take care to use descriptive names for
+variables and classes that adhere to PEP-8 and are consistent with conventions already
+used in PyAEDT.
```
Comment (PipKat) [inline doc/source/Getting_started/Contributing.rst:126]:
```suggestion
Take care to use descriptive names for
variables and classes that adhere to PEP-8 and are consistent with conventions already
used in PyAEDT.
```
Avoid "please" in technical doc.

**Review by PipKat — DISMISSED** [review]
Just some minor nitpicks to adjust content to the Google style guide!

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**